okhttp/build.gradle

```diff
@@ -0,0 +1,17 @@
+// Copyright 2017, Backblaze Inc. All Rights Reserved.
+// License https://www.backblaze.com/using_b2_code.htm
+
+project.ext {
+    pomArtifactId = 'b2-sdk-okhttp'
+    pomName = 'B2 SDK for Java for OkHttp'
+    pomDescription = 'OkHttp support for B2 SDK for Java.'
+}
+
+apply from: '../common.gradle'
+
+dependencies {
+    implementation 'com.backblaze.b2:b2-sdk-core:2.0.0'
+    implementation "com.squareup.okhttp3:okhttp:3.12.1"
+    implementation "com.squareup.okhttp3:okhttp-urlconnection:3.12.1"
+    implementation 'org.jetbrains:annotations:15.0'
```

Comment:
or is okhttp using it?

Reply:
@Nullable etc

Reply:
it looks like maybe we're using @NotNull and @Nullable.  these methods are only being used in this one file and, since they're not adding value to b2-sdk-java users, i'd like to minimize our external dependencies.

how would you feel about removing this dependency?

Reply:
removed